server/tests-py/test_scheduled_triggers.py

```diff
@@ -78,7 +78,10 @@ def test_create_trigger_with_error_returning_webhook(self,hge_ctx):
         assert st == 200, resp
 
     def test_check_fired_webhook_event(self,hge_ctx,scheduled_triggers_evts_webhook):
-        event = scheduled_triggers_evts_webhook.get_event(65)
+        # Ideally, the timeout should be 60 seconds, within which a scheduled event
+        # should be delivered, but this test is flaky while it's run in CI. So,
+        # increasing the timeout to 120 seconds.
+        event = scheduled_triggers_evts_webhook.get_event(120)
```

Comment:
Isn't 120s kind of an overkill. Can we try 70?

Reply:
The 120s here is the max timeout to get the event , but I agree that 120 seems to be an overkill. I'll experiment with lower values starting from 70.

Reply:
Do you know why 65s sometimes doesn't work?

Reply:
We run the tests parallelly in some of the configurations, which may cause the flakiness of the test. I have added a decorator to the tests to run the tests sequentially with timeout as 65. But still not 100% sure if this was the problem.